api/catalog/api/serializers/audio_serializers.py

```diff
@@ -145,6 +145,9 @@ def to_representation(self, instance):
             if not audio.thumbnail:
                 output["thumbnail"] = None
 
+        if isinstance(instance, Audio) and not instance.thumbnail:
+            output["thumbnail"] = None
+
```

Comment:
I can't do a suggestion in the undiffed areas but do you think it might make sense to do something like this? Just so we aren't setting the thumbnail to `None` in every type branch:
```python
def to_representation(self, instance):
    # Get the original representation
    output = super().to_representation(instance)
    audio = instance
    # TODO: Remove when updating ES indexes
    if isinstance(instance, Hit):
        audio = Audio.objects.get(identifier=instance.identifier)

    if isinstance(audio, Audio) and not instance.thumbnail:
        output["thumbnail"] = None

    return output
```